admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -184,7 +184,7 @@ export default class OrderProductRenderer {
   }
 
   toggleProductAddNewInvoiceInfo() {
-    if ($(OrderViewPageMap.productAddInvoiceSelect).val() == 0) {
+    if (0 === $(OrderViewPageMap.productAddInvoiceSelect).val()) {
```

Comment:
Airbnb prefer this style :)
```suggestion
    if ($(OrderViewPageMap.productAddInvoiceSelect).val() === 0) {
```

Reply:
Done, but it's too bad Yoda style is cool and Yoda style is safe ^^